src/Modules/Finance/Header/Header.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { TargetSum } from './TargetSum/TargetSum';

```

Comment:
Instead of using such long paths when importing modules you can do the following:
1. In a component folder create an `index.ts` file (e.g. TargetSum/index.ts)
2. In the `index.ts` export everything you need from TargetSum.tsx component
3. In Header.tsx component the import statement will be like `import { TargetSum } from './TargetSum`
https://medium.com/@balramchavan/smarter-way-to-organize-import-statements-using-index-ts-file-s-in-angular-c685e9d645b7

Reply:
Done